examples/basic_bq/variables.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "dataset_id" {
+  description = "Unique ID for the dataset being provisioned"
+}
+
+variable "dataset_name" {
+  description = "Friendly name for the dataset being provisioned"
+}
+
+variable "description" {
+  description = "Dataset description"
+}
+
+variable "expiration" {
+  description = "TTL of tables using the dataset in MS"
+}
+
+variable "project_id" {
+  description = "Project wheree the dataset and table are created"
+}
+
+variable "table_id" {
+  description = "Unique ID for the table being provisioned"
+}
+
+variable "region" {
+  description = "The regional location for the dataset only US and EU are allowed in module"
+}
+
+variable "time_partitioning" {
+  description = "Configures time-based partitioning for this table"
+}
+
+variable "schema_file" {
+  description = "A JSON schema for the table"
+}
+
+variable "dataset_labels" {
+  description = "Key value pairs in a map for dataset labels"
+  type        = "map"
+}
+
+variable "table_labels" {
+  description = "Key value pairs in a map for table labels"
```

Comment:
Style: the "key value pairs in a map" is implied, since maps are a set of key/value pairs 🙂 We can just use the description from the underlying resource: "A mapping of labels to assign to the table."